src/components/support-form/types/SuportFormData.ts

```diff
@@ -0,0 +1,48 @@
+export type SupportFormData = {
+  terms: boolean
+  info: {
+    email: string
+    name: string
+    phone: string
+    address: string
+  }
+  roles: {
+    benefactor: boolean
+    partner: boolean
+    volunteer: boolean
+    associationMember: boolean
+    promoter: boolean
+  }
```

Comment:
Here's an idea about the types of roles

```suggestion
type RoleTypes = 'benefactor' | 'partner' | 'volunteer' | 'associationMember' | 'promoter'

export type SupportFormData = {
  terms: boolean
  info: {
    email: string
    name: string
    phone: string
    address: string
  }
  roles: { [key in RoleTypes]: boolean }
```

When we split a large type into smaller types it becomes easier to track the changes, reuse the types and see the whole picture